src/lib.rs.in

```diff
@@ -719,6 +719,33 @@ class Duration{
     pre_build_generate_equals_and_hashcode Duration;
 });
 
+foreign_class!(
+class EncryptedBlindIndexSalt {
```

Comment:
I think we need this type to have a constructor that takes the deks and data.

We should also have a test that does this to show how it works.

Reply:
Done (I think)